### DISCUSSION TITLE
Adjusting DT to make a map faster than 1.5 should not reset the multiplier to x1.0

### DISCUSSION THREAD

**Discussion by kissinglucifer** [ppy/osu#21863] [Ideas] [answered]
Currently using dt on speed 1.75 will reset the multiplier to x1.0 compared to x1.12. In my opinion the mod should just keep the x1.12 multiplier. ~~and maybe change other mods to work like this?? idk~~

**Comment by FlamingSquid** [comment]
I believe that this is a temporary thing that has to do with the new implementation of the pp system. I think that changing mod settings will be supported sometime in the future.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/19711
